transport/awssqs/consumer.go

```diff
@@ -0,0 +1,242 @@
+package awssqs
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/request"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	"github.com/go-kit/kit/endpoint"
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/transport"
+)
+
+// Consumer wraps an endpoint and provides and provides a handler for sqs msgs
```

Comment:
Please format all doc comments as full English sentences with punctuation and proper initialism capitalization, e.g.

```suggestion
// Consumer wraps an endpoint and provides a handler for SQS messages.
```

This comment applies to all doc comments in the PR.